modules/templates/helper.go

```diff
@@ -88,6 +88,9 @@ func NewFuncMap() []template.FuncMap {
 		"AllowedReactions": func() []string {
 			return setting.UI.Reactions
 		},
+		"ManifestData": func() template.URL {
```

Comment:
Maybe we should set this as a global var but not use a function. I think Golang will not optimize here.

Reply:
Hmm sure if I find out how. I guess the same goes for the other `setting` variables above as well.

Reply:
I'm not sure where we'd add such variables. `ExecuteTemplate` is only called in macaron renderer and that one does not seem to allow additional static variables on instantiation (https://godoc.org/gopkg.in/macaron.v1#RenderOptions).

Reply:
It's not obvious. It's in https://github.com/go-gitea/gitea/blob/master/modules/context/context.go#L346. And you are right. All inline functions should be moved there.

Reply:
Indeed, but let's do that in another PR.

Reply:
Did it for this variable now.

Reply:
Thought I have to say there is one benefit of using a helper vs. a variable. Helper can be used anywhere easily, even withing a `range` subtemplate which would otherwise require awkward passing of data using the `dict` helper. So I think it's fine to do in this case, but may not be in others.

Reply:
Hm, you are right. No global variables but global functions on golang template.